torch/nn/modules/transformer.py

```diff
@@ -158,34 +159,36 @@ def forward(self, src: Tensor, tgt: Tensor, src_mask: Optional[Tensor] = None, t
                 forward and backward compatibility.
 
         Shape:
-            - src: :math:`(S, E)` for unbatched input, :math:`(S, N, E)` if `batch_first=False` or
-              `(N, S, E)` if `batch_first=True`.
-            - tgt: :math:`(T, E)` for unbatched input, :math:`(T, N, E)` if `batch_first=False` or
-              `(N, T, E)` if `batch_first=True`.
+            - src: :math:`(S, E)` for unbatched input, :math:`(S, N, E)` if ``batch_first=False`` or
+              :math:`(N, S, E)` if ``batch_first=True``.
+            - tgt: :math:`(T, E)` for unbatched input, :math:`(T, N, E)` if ``batch_first=False`` or
+              :math:`(N, T, E)` if ``batch_first=True``.
             - src_mask: :math:`(S, S)` or :math:`(N\cdot\text{num\_heads}, S, S)`.
             - tgt_mask: :math:`(T, T)` or :math:`(N\cdot\text{num\_heads}, T, T)`.
             - memory_mask: :math:`(T, S)`.
             - src_key_padding_mask: :math:`(S)` for unbatched input otherwise :math:`(N, S)`.
             - tgt_key_padding_mask: :math:`(T)` for unbatched input otherwise :math:`(N, T)`.
             - memory_key_padding_mask: :math:`(S)` for unbatched input otherwise :math:`(N, S)`.
 
-            Note: [src/tgt/memory]_mask ensures that position i is allowed to attend the unmasked
-            positions. If a BoolTensor is provided, positions with ``True``
-            are not allowed to attend while ``False`` values will be unchanged. If a FloatTensor
-            is provided, it will be added to the attention weight.
-            [src/tgt/memory]_key_padding_mask provides specified elements in the key to be ignored by
-            the attention. If a BoolTensor is provided, the positions with the
-            value of ``True`` will be ignored while the position with the value of ``False`` will be unchanged.
+                ..note::
+                    [src/tgt/memory]_mask ensures that position i is allowed to attend the unmasked
```

Comment:
```suggestion
                    [src/tgt/memory]_mask ensures that position i is allowed to attend to the unmasked
```
this grammar feels more right to be but ¯\_(ツ)_/¯

Reply:
also, math i?